user/serializers.py

```diff
@@ -1 +1,61 @@
-# write your code here
+from django.contrib.auth import get_user_model
+from django.contrib.auth.password_validation import validate_password
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = get_user_model()
+        fields = ("id", "username", "email", "password", "is_staff")
+        read_only_fields = ("id", "is_staff")
+        extra_kwargs = {
+            "password": {"write_only": True, "min_length": 5}
+        }
+
+    def create(self, validated_data):
+        user = get_user_model().objects.create_user(**validated_data)
```

Comment:
Consider using `create_user` method from the user model to ensure that the password is hashed and any additional user creation logic is applied. This is already done here, but it's important to highlight its necessity for security.